docs/api/QUIC_CONNECTION_EVENT.md

```diff
@@ -176,6 +176,8 @@ Value | Meaning
 **QUIC_STREAM_OPEN_FLAG_UNIDIRECTIONAL**<br>1 | A unidirectional stream.
 **QUIC_STREAM_OPEN_FLAG_0_RTT**<br>2 | The stream was received in 0-RTT.
 
+If a server wishes to use `QUIC_STREAM_OPEN_FLAG_DELAY_FC_UPDATES` for the newly started stream, it may append this flag to `Flags` before it returns from the callback.
```

Comment:
Is this flag for stream ID flow control or stream flow control?

Reply:
The entire PR is for stream ID. I think it should be better renamed to QUIC_STREAM_OPEN_FLAG_DELAY_ID_FC_UPDATES

Reply:
Ok, I will update.